src/volcano-app.js

```diff
@@ -1,76 +1,65 @@
 (function() {
   'use strict';
 
-  class VolcanoApp extends Polymer.Element {
-    static get is () { return 'volcano-app'; }
+  let defaultPage = 'view1';
 
-    static get config () {
-      return {
-        properties: {
-          page: {
-            type: String,
-            reflectToAttribute: true,
-            observer: '_pageChanged'
-          }
-        },
+  Polymer({
+    is: 'volcano-app',
 
-        observers: [
-          '_routePageChanged(routeData.page)'
-        ],
-      };
-    }
-
-    constructor () {
-      super();
-      this.defaultPage = 'view1';
-    }
+    properties: {
+      page: {
+        type: String,
+        reflectToAttribute: true,
+        observer: '_pageChanged'
+      }
+    },
 
-    ready () {
-      super.ready();
+    observers: [
+      '_routePageChanged(routeData.page)'
+    ],
 
+    ready: function () {
       //remove frist loading style
       this.removeAttribute('unresolved');
 
       this._setHeaderScrollListener(this.$.appHeader, this._onHeaderScroll);
-    }
+    },
 
-    _setHeaderScrollListener(header, fn) {
+    _setHeaderScrollListener: function (header, fn) {
       let target = (header.scrollTarget === this.ownerDocument.documentElement) ? window : header.scrollTarget;
       target.addEventListener('scroll', fn.bind(this), { passive: true });
       //TODO: add passive polyfill
-    }
+    },
 
-    _onHeaderScroll () {
+    _onHeaderScroll: function () {
       let y = window.scrollY;
       y *= 1.5;
 
       // header background scroll effect
```

Comment:
in opera work but still not work on chrome!
we don't have header shadow in chrome (In linux environment )